index.html

```diff
@@ -0,0 +1,697 @@
+            SPONSORED INSIGHT
+          </figcaption>
+        </figure>
+        <a class="mb-2 p-1 font-weight-bolder text-body h5" href="#">What Makes International Schools so Special?</a>
+        <p class="pl-1">One of the single most important decisions you make for your child is whom to entrust with their education. See why so many parents are turning to International Schools.</p>
+      </div>
+      <div class="d-md-block order-last order-md-last card border-1 mb-4">
+        <figure class="position-relative">
+          <img src="assets/sponsor3.png" alt="showcase-image" class="w-100" />
+          <figcaption class="position-absolute fixed-bottom w-40 pl-1 d-flex justify-content-center align-items-center bg-light font-weight-bolder text-primary">
+            RANKINGS
+          </figcaption>
+        </figure>
+        <a class="mb-2 p-1 font-weight-bolder text-body h5" href="#">America's Most Responsible Companies 2021</a>
+        <p class="pl-1">We partnered with Statista to rank the country's Most Responsible Companies. They share the values that we all prioritize, and that's worth recognizing in 2021</p>
+      </div>
+      <div class="d-md-block order-last order-md-last card border-1 mb-4">
+        <figure class="position-relative">
+          <img src="assets/sponsor.jpg" alt="showcase-image" class="w-100" />
+          <figcaption class="position-absolute fixed-bottom w-40 pl-1 d-flex justify-content-center align-items-center bg-light font-weight-bolder text-primary">
+            SPONSORED INSIGHT
+          </figcaption>
+        </figure>
+        <a class="mb-2 p-1 font-weight-bolder text-body h5" href="#">Hokkaido: Japan’s Northern Frontier</a>
+        <p class="pl-1">The Biden Administration will issue an order stopping construction of the border wall on its first day.
+        </p>
+      </div>
+    </div>
+
+  </section>
+
+<br>
+
+<section style="font-family: Arial, Helvetica, sans-serif;"
+class="bg-white row border-top border-bottom pt-5 justify-content-between flex-md-row flex-column footer-social">
+<div class="w-100 w-md-0 d-md-flex justify-content-start col-md-5">
+  <img class="img-fluid w-50 p-0 justify-content-start mb-3 footer-logo" src="assets/logo-red.svg" alt="Red logo">
+  <p class='col-md-2 text-nowrap align-self-center text-left mb-md-n2 ml-md-2 p-0  mb-2 text-muted'>© 2021 NEWSWEEK
+    DIGITAL LLC</p>
+</div>
+<ul class="list-unstyled row col-md-4 m-0 mb-3 justify-content-md-end">
+  <li><i class="fab fa-facebook-square mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-twitter mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-instagram mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-tumblr mx-1 fa-2x text-muted"></i></li>
+  <li><i class="fab fa-linkedin-in mx-1 fa-2x text-muted"></i></li>
+</ul>
+</section>
+
+<footer style="font-family: Arial, Helvetica, sans-serif;" class="pt-5 bg-white row">
```

Comment:
- [ ] Please don't include inline stylings, instead put all of this in the CSS file.